frontend/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "online-judge-frontend",
+  "version": "1.0.0",
+  "description": "Frontend for the online-judge made by CL",
+  "main": "index.js",
+  "scripts": {
+    "dev": "next",
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "lint": "eslint ./shared ./components ./pages --fix"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Cyber-Labs/cl-judge.git"
+  },
+  "author": "Cyber Labs",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/Cyber-Labs/cl-judge/issues"
+  },
+  "homepage": "https://github.com/Cyber-Labs/cl-judge#readme",
+  "dependencies": {
+    "bootstrap": "^4.5.2",
+    "font-awesome": "^4.7.0",
+    "formik": "^2.1.6",
+    "next": "^9.5.3",
+    "react": "^16.13.1",
+    "react-bootstrap": "^1.3.0",
+    "react-dom": "^16.13.1",
+    "yup": "^0.29.3"
+  },
+  "devDependencies": {
+    "eslint": "^7.10.0",
+    "eslint-config-prettier": "^6.12.0",
+    "eslint-config-standard": "^14.1.1",
+    "eslint-plugin-jsx-a11y": "^6.3.1",
+    "eslint-plugin-prettier": "^3.1.4",
+    "eslint-plugin-react": "^7.21.2",
+    "eslint-plugin-react-hooks": "^4.1.2",
+    "eslint-plugin-standard": "^4.0.1"
```

Comment:
the prettier extension is not there.....for semicolons and single quotes
that's why error is there

Reply:
```
"eslint": "^6.1.0",
"eslint-config-airbnb": "^18.0.1",
"eslint-config-prettier": "^6.0.0",
"eslint-config-standard": "^13.0.1",
"eslint-plugin-import": "^2.18.2",
"eslint-plugin-jsx-a11y": "^6.2.3",
"eslint-plugin-node": "^9.1.0",
"eslint-plugin-prettier": "^3.1.0",
"eslint-plugin-promise": "^4.2.1",
"eslint-plugin-react": "^7.14.3",
"eslint-plugin-react-hooks": "^1.7.0",
"eslint-plugin-standard": "^4.0.0",
"prettier": "1.18.2"
```

Reply:
actually it was due to the style setup. I used 'standard' style and now it  is fixed.

Reply:
ok